src/python_routing_v02/troute/routing/fast_reach/diffusive.pyx

```diff
@@ -273,21 +273,25 @@ cpdef object compute_diffusive_tst(
     cdef int gages_size = usgs_positions.shape[0]
     cdef int gage_maxtimestep = usgs_values.shape[1]
     cdef int gage_i, usgs_position_i, timestep
-    cdef float lastosbs_value, lastobs_time
+    cdef float[:] lastosbs_values, lastobs_times
```

Comment:
This looks like it may be a spelling error...

Reply:
Sure is. Do you mind fixing and committing?